README.md

```diff
@@ -1,7 +1,16 @@
-## GraphQL Client
+# GraphQL Client
 
 This is a simple graphQL client which can be used by other TDR projects to communicate with the consignment API and any other future graphQL APIs.
 
 [GraphQLService](src/main/scala/uk/gov/nationalarchives/tdr/GraphQLService.scala) contains a single method to get a client.
 [GraphQLClient](src/main/scala/uk/gov/nationalarchives/tdr/GraphQLClient.scala) contains the logic for the client. It takes generated case classes as arguments which are found in the [generated graphql](https://github.com/nationalarchives/tdr-generated-graphql) project.
 
+## Local development
+
+To publish the library locally, run:
+
+```
+sbt +package +publishLocal
+```
+
```

Comment:
We don't need the + any more to publish multiple versions since the front end was switched over to 2.13. There's a small task to change this build.sbt to remove the 2.12 stuff but seeing as 2.13 is the default, removing the + sign works for now.

Reply:
It defaulted to 2.12 when I just ran `sbt package publishLocal`, which is why I needed to add the + signs.

I can remove the old 2.12 config if it's no longer needed, but do you think we should leave the instructions with the + sign? We'll eventually have to upgrade to 2.14, etc, and we'll probably have multiple apps using this client by then so they won't all be done at once.

Reply:
Yeah ok, fair point, I thought it was defaulting to 2.13. I think I'm thinking of something else.